datastore/csv.go

```diff
@@ -0,0 +1,28 @@
+package datastore
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+)
+
+func ReadCsvFile(filePath string) [][]string {
```

Comment:
By convention, all the exported stuff should have a description.
```go
// ReadCsvFile - Description here
func ReadCsvFile(filePath string) [][]string {
```
Do it for all the exported stuf.